openbook/urls.py

```diff
@@ -260,7 +260,7 @@
     path('avatar/', CommunityAvatar.as_view(), name='community-avatar'),
     path('cover/', CommunityCover.as_view(), name='community-cover'),
     path('favorite/', FavoriteCommunity.as_view(), name='favorite-community'),
-    path('subscribe/', SubscribeCommunity.as_view(), name='subscribe-community'),
+    path('subscribe/notifications/', SubscribeCommunityNotifications.as_view(), name='subscribe-community-notifications'),
```

Comment:
notifications/subscribe and `SubscribeToCommunityNotifications` for the APIView please 🙌 , if we ever have something related to community subscriptions, better to have this namespace available.